src/main/java/carracing/domain/Winner.java

```diff
@@ -0,0 +1,28 @@
+package carracing.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class Winner {
+    private List<Car> cars;
+
+    public Winner(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    protected String getWinner() {
+        int maxPosition = getMaxPosition();
+        return cars.stream()
+                .filter(car -> maxPosition <= car.getPosition())
```

Comment:
```suggestion
                .filter(car -> car.isMaxPosition(maxPosition))
```
와 같이 car 도메인에게 메시지를 전달하는 구조는 어떠신가요 ?

Reply:
메세지 전달! 적용했습니다~